lmdeploy/pytorch/engine/model_agent.py

```diff
@@ -886,6 +899,103 @@ def release(self):
         torch.cuda.empty_cache()
 
 
+class DefaultForwardInputsMaker:
+    """Default forward inputs maker."""
+
+    def __init__(self, model_agent: BaseModelAgent):
+        self._in_que = model_agent._in_que
+
+    async def get(self):
+        """get."""
+        return await self._in_que.get()
+
+    def step(self):
+        """step."""
+        # No-op for default maker
+        pass
+
+
+class DPForwardInputsMaker:
```

Comment:
What's the benefit of this input maker?

Reply:
Make dummy input to synchronize behaivour of all dp ranks.